app/.env.sample

```diff
@@ -1,2 +1,3 @@
-# .env
-DISCORD_TOKEN={your-bot-token}
+# .env
+DISCORD_TOKEN={your-bot-token}
+DISCORD_USER={user-id of the user to be checked for mentions}
```

Comment:
What is this? The Discord_user id of the author or the user of the bot?

So that you can verify if he/she is being mentioned?

Reply:
its the user-id of the user of the bot, to verify that he/she is mentioned.